### PR TITLE
Enable ZIMPL on osx-arm64

### PR DESCRIPTION
**Pull request by dg0yt**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,6 +28,8 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake >=3.15
     - make            # [unix]
+    - bison           # [unix]
+    - flex            # [unix]
```
Comment (AntoinePrv) [inline recipe/meta.yaml:32]: Bison and flex are libraries or build tools? This section is reserved for build tools.
If a dependency is only needed at build time (e.g. a single header used privately inside a `.c` file), it should still remain in the `host` section.

Comment (AntoinePrv) [reply recipe/meta.yaml:32]: Looking at a few recipe it seems bison is often used in `build` dependency.

**Review by AntoinePrv — APPROVED** [review]

**Comment by AntoinePrv** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (dg0yt) [reply recipe/meta.yaml:32]: bison and flex are build tools.

**Pull request merged**